Firebase/Auth/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# v6.5.0
+- [feature] Added support of Multi-factor Authentication. (#4823)
```

Comment:
@morganchen12 should this be `feature` or `added`?

Reply:
Seems we used both before. My understanding is we use `feature` for minor releases? I'll let Morgan to decide.

Reply:
@ryanwilson Talked to Morgan offline. Actually tehy're the same and they both show up as feature on devsite.